### ISSUE TITLE
Were you able to reproduce results?

### ISSUE THREAD

**Issue by raotnameh** [sooftware/jasper#1]
HI, can you also share the training statistics? That would be very helpful.

**Comment by sooftware** [comment]
I checked that the model converges. But I couldn't afford to train it around for a long time.

**Comment by raotnameh** [comment]
@sooftware For how many number of epochs did you check for? and on what dataset?

**Comment by sooftware** [comment]
KsponSpeech which contains 1,000 hours of Korean speech.  
I experiment with 7~8 epochs.

**Comment by raotnameh** [comment]
@sooftware Did you test and see the WER? or just trained it? If you did the test, then what was the WER you were getting?

**Comment by sooftware** [comment]
Korean usually calculate CER. I got a 20-30 CER. loss was continuing to converge.

**Comment by raotnameh** [comment]
@sooftware Thanks for the update.

**Comment by raotnameh** [comment]
@sooftware can you share your training script and the hyperparameters?

**Comment by sooftware** [comment]
Refer [this repository](https://github.com/sooftware/KoSpeech)

**Comment by raotnameh** [comment]
@sooftware Thanks.
